extensions/amp-a4a/rtc-documentation.md

```diff
@@ -149,6 +149,7 @@ The `errorReportingUrl` property is optional. The only available macros are ERRO
 -   [PubWise](https://www.pubwise.io)
 -   Purch
 -   Rubicon
+-   RichAudience
```

Comment:
Nit: please sort alphabetically (before Rubicon).

Reply:
the change has been applied in the next commit. Please check it.
Cheers